Dockerfile

```diff
@@ -19,12 +19,14 @@ RUN \
     groupadd --gid 10001 app && \
     useradd --uid 10001 --gid 10001 --home /app --create-home app && \
     apt-get -q update && \
-    apt-get -q install -y --no-install-recommends default-libmysqlclient-dev libssl-dev ca-certificates libcurl4 python3.7 python3-pip && \
+    apt-get -q install -y --no-install-recommends default-libmysqlclient-dev libssl-dev ca-certificates libcurl4 python3-venv python3-pip && \
+    python3 -m pip install setuptools wheel && \
+    python3 -m pip install google-cloud-spanner && \
```

Comment:
I'd have preferred all these on a single `pip install` line, but looks like the spanner install fails if I do that as the other required deps don't seem to finish in time.

Reply:
(also - would a `requirements.txt` file be preferred here?) Seems like maybe it'd be cleaner.

Reply:
Huh, I get the same problem when I try to install via requirements.txt